pkg/artifactory/datasource/datasource_artifactory_group.go

```diff
@@ -0,0 +1,47 @@
+package datasource
+
+import (
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/security"
+	"github.com/jfrog/terraform-provider-shared/packer"
+	"github.com/jfrog/terraform-provider-shared/predicate"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"golang.org/x/net/context"
+)
+
+func ArtifactoryGroup() *schema.Resource {
+	dataSourceGroupRead := func(_ context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+		group := security.Group{}
+		name := d.Get("name").(string)
+		includeUsers := d.Get("include_users").(string)
+		m.(*resty.Client).R().SetResult(&group).SetQueryParam("includeUsers", includeUsers).Get(security.GroupsEndpoint + name)
+
+		d.SetId(group.Name)
+
+		pkr := packer.Universal(predicate.SchemaHasKey(security.GroupSchema))
+		return diag.FromErr(pkr(&group, d))
+	}
+
+	getDataSourceGroupSchema := func() map[string]*schema.Schema {
+		var includeUsersSchema = map[string]*schema.Schema{
```

Comment:
Not a blocker. In future, let's be consistent with style. Either use `:=` or `var`, and not mix them. (Unless the compiler forces your to 😄 )